### DISCUSSION TITLE
Can't access DragEvent properties

### DISCUSSION THREAD

**Discussion by mattoni** [leptos-rs/leptos#2397] [Q&A] [answered]
Per the recommendation of adding a web-sys dependency with the requested features, I added `web-sys = { version = "0.3.68", features = ["DragEvent"] }` to my Cargo.toml, but am still unable to access the `data_transfer()` property of the event. Any suggestions?

![image](https://github.com/leptos-rs/leptos/assets/5110855/5f53d227-6477-4f58-acfa-7ab0db102d06)

![image](https://github.com/leptos-rs/leptos/assets/5110855/367aef6e-4a67-4680-86a1-4283f1b4f8d1)

**Comment by gbj** [comment, marked as answer]
It looks like `.data_transfer` [requires the `DragEvent` and `DataTransfer` features](https://docs.rs/web-sys/latest/web_sys/struct.DragEvent.html#method.data_transfer). `leptos_dom` already enables `DragEvent`, you are just missing `DataTransfer`.

**Reply by mattoni** [reply]
That did it thanks!
